packages/babel-register/test/experimental-loader.js

```diff
@@ -0,0 +1,47 @@
+import { spawnSync } from "child_process";
+import { dirname, join } from "path";
+import { fileURLToPath } from "url";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const [major] = process.versions.node.split(".").map(Number);
+
+function runTest(fixture, args, output) {
+  const out = spawnSync(
+    "node",
+    ["--experimental-loader", join(__dirname, "..", "loader.mjs"), ...args],
+    {
+      stdio: "pipe",
+      env: {
+        ...process.env,
+        BABEL_REGISTER: join(__dirname, "fixtures", fixture, "register.mjs"),
+      },
+      cwd: join(__dirname, "fixtures", fixture),
+    },
+  );
+
+  expect(String(out.stderr).split("\n").slice(2).join("\n")).toEqual("");
+  expect(out.error).toBeUndefined();
+  expect(out.signal).toBeNull();
+  expect(String(out.stdout)).toEqual(output);
+}
+
+// skip everything if below version 12
+if (major < 12) {
+  // eslint-disable-next-line no-global-assign
+  it = it.skip;
```

Comment:
Let's use a new variable.

Reply:
ide doesn't detect it as mocha test then, for running it from ui